certbot-ci/certbot_integration_tests/conftest.py

```diff
@@ -89,10 +89,11 @@ def _setup_primary_node(config):
                              'boulder, but is not installed or not available for current user.')
 
         try:
-            subprocess.check_output(['docker-compose', '-v'], stderr=subprocess.STDOUT)
+            subprocess.check_output(['docker', 'compose', 'ps'], stderr=subprocess.STDOUT)
```

Comment:
looks like the test run i kicked off failed. maybe try `docker compose ls` here? `docker compose ps` fails for me locally if there's not a `docker-compose.yml` file in the current working directory